package.json

```diff
@@ -20,59 +20,62 @@
+    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
+    "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
     "lint:hbs": "ember-template-lint .",
     "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "npm-run-all lint test:*",
+    "test": "concurrently \"npm:lint\" \"npm:test:*\" --names \"lint,test:\"",
     "test:ember": "ember test",
     "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.23.1",
+    "ember-cli-babel": "^7.26.11",
     "git-repo-info": "^2.1.1"
   },
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^2.2.9",
-    "@glimmer/component": "^1.0.3",
-    "@glimmer/tracking": "^1.0.3",
+    "@ember/test-helpers": "^2.8.1",
+    "@embroider/test-setup": "^2.0.2",
+    "@glimmer/component": "^1.1.2",
+    "@glimmer/tracking": "^1.1.2",
     "babel-eslint": "^10.1.0",
-    "ember-auto-import": "^1.11.3",
-    "ember-cli": "~3.25.2",
-    "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-htmlbars": "^5.3.2",
+    "concurrently": "^7.6.0",
+    "ember-auto-import": "^2.5.0",
+    "ember-cli": "~4.9.2",
+    "ember-cli-dependency-checker": "^3.3.1",
+    "ember-cli-htmlbars": "^6.1.1",
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-terser": "^4.0.2",
-    "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-export-application-global": "^2.0.1",
     "ember-load-initializers": "^2.1.2",
-    "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-page-title": "^6.2.2",
-    "ember-qunit": "^5.1.4",
-    "ember-resolver": "^8.0.2",
-    "ember-source": "3.25.3",
+    "ember-page-title": "^7.0.0",
+    "ember-qunit": "^6.0.0",
+    "ember-resolver": "^8.0.3",
+    "ember-source": "4.9.1",
     "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^2.18.1",
+    "ember-template-lint": "^5.2.0",
     "ember-try": "^1.4.0",
     "eslint": "^7.32.0",
-    "eslint-config-prettier": "^7.2.0",
-    "eslint-plugin-ember": "^10.5.1",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-ember": "^11.2.1",
     "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.4.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-qunit": "^7.3.4",
     "loader.js": "^4.7.0",
-    "npm-run-all": "^4.1.5",
-    "prettier": "^2.3.2",
-    "qunit": "^2.16.0",
-    "qunit-dom": "^1.6.0",
+    "prettier": "^2.8.1",
+    "qunit": "^2.19.3",
+    "qunit-dom": "^2.0.0",
     "release-it": "^14.10.1",
-    "release-it-lerna-changelog": "^3.1.0"
+    "release-it-lerna-changelog": "^3.1.0",
+    "webpack": "^5.75.0"
+  },
+  "peerDependencies": {
+    "ember-source": "^3.28.0 || ^4.0.0"
   },
   "engines": {
-    "node": "10.* || >= 12"
+    "node": "14.* || 16.* || >= 18"
```

Comment:
This would be a breaking change, so best to do this in a separate PR.

Reply:
I was thinking this PR would be the breaking change PR

- Drop support for Node < 14
- Drop support for Ember < 3.28

[v5.0.0](https://github.com/ember-cli/ember-cli-app-version/releases/tag/v5.0.0) was done in a similar way

Reply:
Doing a blueprint update, dropping support for Ember versions and dropping support for Node versions are unrelated things, that's why I would've preferred doing these things in separate PRs. Then we would also have separate changelog entries pointing to separate PRs, which seems cleaner to me.

Reply:
They seem related to me. My understanding is that blueprints are paired with specific Node and Ember versions, i.e they're part of the blueprint and were automatically updated by `ember-cli-update` in this case.

Reply:
A specific version of the blueprint indeed comes with a specific Ember and node support matrix, but that doesn't mean we must drop support for Ember and Node versions to do an internal blueprint update I feel. If we make separate / specific pull requests, it will be easier to generate a clear changelog.

Reply:
have the ember-cli team considered not requiring specific sets of PRs for automated changelog generation?

it's kind of prohibitive to contributions when _both_ changes need/should happen anyway.

If we move to something like Changesets, we can have _much_ better changelogs:
Some examples:
 - https://github.com/CrowdStrike/ember-headless-table/blob/main/ember-headless-table/CHANGELOG.md
 - With a preview PR: https://github.com/NullVoxPopuli/ember-resources/pull/686 so folks can see what will be in the next release
 
 This moves the responsibility of managing the changelog from contributors to the repo owners, where it should be.
 
 Convention-based changelogs deter contributors, and I don't think we can afford that :sweat_smile:

Reply:
:thinking: do you have thoughts on then forbidding ember-cli-update from running on addons? -- we could constrain it to do so. It seems "git history" and _the recommended way to update ember stuff_ are in conflict.

But regardless of it's the better thing to do, the work is done, and it's work we want in totality anyway, so... _why split it further_?

Consumers of this addon don't really care about the git history, and we're generally all familiar with blueprint updates.